src/aero_kernel/src/userland/mod.rs

```diff
@@ -27,10 +27,10 @@ pub mod task;
 pub mod vm;
 
 pub fn run() -> fs::Result<()> {
-    let shell_path = Path::new("/bin/aero_shell");
-    let shell_inode = fs::lookup_path(shell_path)?;
+    let init_path = Path::new("/bin/aero_init");
```

Comment:
Hmm, for simplicity maybe not prefix the init binary with `aero` since for aero_shell it makes sense but init is more general ig?

Reply:
Renamed in 3b026c8ee592014ca30c9981ce64b767e12edddd